stress-example/image_search/app.py

```diff
@@ -61,31 +29,33 @@ def config(indexer_query_type):
     elif indexer_query_type == 'scann':
         os.environ['JINA_USES'] = os.environ.get('JINA_USES_SCANN', 'docker://scann_indexer_image:test')
         os.environ['JINA_USES_INTERNAL'] = 'pods/scann_indexer.yml'
-
-
-docs_index = random_docs(0, NUM_DOCS)
-docs_query = random_docs(0, QUERY_NUM_DOCS)
+    elif task == 'query':
+        raise ValueError(f'Indexer type {indexer_query_type} not supported')
 
 
 # for index
 def index():
     flow_index = Flow.load_config('flows/index.yml')
```

Comment:
can u set ? I think this should work
```python
Flow.load_config('flows/index.yml', port_expose=os.environ['JINA_PORT'])
```

Reply:
It wasn't set in the `index.yml` Flow. I added it and now it uses that